### DISCUSSION TITLE
Dbeaver Column Description

### DISCUSSION THREAD

**Discussion by MrLastOne** [dbeaver/dbeaver#21701] [General]
Hello,

Sorry for the new discussion, but during search haven't found anything that would help. Have purchased new laptop, installed new version of DBeaver and once connected to the database now not able to see actual column description when hovering over columns name. Example attached. Any idea what I'm missing (DB2 for IBM i)?

![DBeaver diff](https://github.com/dbeaver/dbeaver/assets/149401535/e7406989-a306-4328-b089-25bb6f8dca0e)

Thank you.

**Comment by LonwoLonwo** [comment]
Hello. These settings may help.

![2023-11-01 10_57_25-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/e144731d-5163-45d5-89f9-123bee3fe489)

![2023-11-01 10_58_47-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/f1c21a84-9df0-4258-ba4d-ae9066af4ac8)

Also, maybe this topic will be helpful: https://github.com/dbeaver/dbeaver/issues/13335
Or other db2i tickets: https://github.com/dbeaver/dbeaver/issues?q=is%3Aopen+is%3Aissue+label%3Ax%3Adb2i

**Reply by MrLastOne** [reply]
Thank you, after reading topic found a solution, in my case setting up metadata source to 0 did the trick.
